src/Rasterization/Renderers/SVGRectRenderer.php

```diff
@@ -86,4 +156,76 @@ protected function renderStroke($image, array $params, $color, $strokeWidth)
             $color
         );
     }
+
+    private function renderStrokeRounded($image, array $params, $color, $strokeWidth) {
+        $x1 = $params['x1'];
+        $y1 = $params['y1'];
+        $x2 = $params['x2'];
+        $y2 = $params['y2'];
+        $rx = $params['rx'];
+        $ry = $params['ry'];
+
+        $halfStrokeFloor = floor($strokeWidth / 2);
+        $halfStrokeCeil  = ceil($strokeWidth / 2);
+
+        // top
+        imagefilledrectangle(
+            $image,
+            $x1 - $halfStrokeFloor + $rx*1.5,  $y1 - $halfStrokeFloor,
+            $x2 + $halfStrokeFloor - $rx*1.5,  $y1 + $halfStrokeCeil - 1,
+            $color
+        );
+        // bottom
+        imagefilledrectangle(
+            $image,
+            $x1 - $halfStrokeFloor + $rx*1.5,  $y2 - $halfStrokeCeil + 1,
+            $x2 + $halfStrokeFloor - $rx*1.5,  $y2 + $halfStrokeFloor,
+            $color
+        );
+        // left
+        imagefilledrectangle(
+            $image,
+            $x1 - $halfStrokeFloor,     $y1 + $halfStrokeCeil + $ry/2,
+            $x1 + $halfStrokeCeil - 1,  $y2 - $halfStrokeCeil - $ry/2,
+            $color
+        );
+        // right
+        imagefilledrectangle(
+            $image,
+            $x2 - $halfStrokeCeil + 1,  $y1 + $halfStrokeCeil + $ry/2,
+            $x2 + $halfStrokeFloor,     $y2 - $halfStrokeCeil - $ry/2,
+            $color
+        );
+        imagesetthickness($image, 1.5);
+        for ($sw = $strokeWidth  ; $sw >= -$halfStrokeCeil; $sw --) {
```

Comment:
I want to use current style, but, imagearc is not good at thickness > 1.
```
$im = imagecreatetruecolor(150, 150);
imagesetthickness($im, 30);
imagearc($im, 100, 100, 100, 100, 180, 270, 0x00ff00ff);
imagepng($im);
```
![image](https://user-images.githubusercontent.com/26040/29648369-b2aff036-88c8-11e7-82fa-5a6c6f7da017.png)